components/automate-cs-ocid/habitat/default.toml

```diff
@@ -39,7 +39,7 @@ pool_init_size=5
 pool_queue_max=50
 
 [ocid]
-  [chef_server_config]
+  [ocid.chef_server_config]
```

Comment:
This is done in the feature branch for the demo purpose. You can skip this change.

Reply:
I will manage it during rebase if required.